src/ImagineSvg/Image.php

```diff
@@ -0,0 +1,461 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\ImagineSvg;
```

Comment:
Correct. They are an implementation of the Imagine interfaces. We can split them into its own package some time.